nullaway/src/test/java/com/uber/nullaway/NullAwayContractsTests.java

```diff
@@ -64,6 +64,10 @@ public void basicContractAnnotation() {
             "    NullnessChecker.assertNonNull(o);",
             "    return o.toString();",
             "  }",
+            "  String test4(java.util.Map<String,Object> m) {",
+            "    NullnessChecker.assertNonNull(m.get(\"foo\"));",
+            "    return m.get(\"foo\").toString();",
```

Comment:
Cool! Just ran into basically this case internally (not worth cutting a new release, I think, but this would have saved us at least one `castToNonNull`)